packages/vm/tests/BlockchainTestsRunner.js

```diff
@@ -28,7 +28,7 @@ module.exports = async function runBlockchainTest(options, testData, t) {
 
   const blockchain = new Blockchain({
     db: blockchainDB,
-    hardfork,
+    common: options.common,
```

Comment:
small style nit: instead of setting `{ common: options.common }` multiple times here and below you can do `const { common } = options` once then simply pass `{ common }` every time like i did with `hardfork`. no big deal though :)

Reply:
Good point, thanks 😄